fake/fake-extension.js

```diff
@@ -830,17 +825,20 @@ var generateMvcRequest = (function() {
                     
                     var payload = message.payload;
                     if (type == 'begin-request') {
-                        request._requestStartTime = payload.requestStartTime;
-                        request._requestMethod = payload.requestMethod;
-                        request._requestUrl = (payload.requestPath || '') + (payload.requestQueryString || '');
+                        payload.path = source.path;
+                        payload.query = source.query;
+                        
```

Comment:
Correct. This is needed for internal purposes. The client adds additional derived data.